app/javascript/components/shared/modals/DeleteRoomModal.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import DeleteRoomForm from '../../forms/DeleteRoomForm';
+import Modal from '../Modal';
+
+export default function DeleteRoomModal({ mutation, title, modalButton }) {
+  return (
+    <Modal
+      modalButton={modalButton}
+      title={title}
+      body={<DeleteRoomForm mutation={mutation} />}
+    />
+  );
+}
```

Comment:
Having a full component just for this seems odd to me. Why not just call `Modal` wherever we're using it?

Reply:
Yup, you got reason.

Reply:
Was thinking about further simplifying the re-use of some Modals, but it's an overkill